soc/arm/xilinx_zynqmp/soc.c

```diff
@@ -29,3 +28,12 @@ static int soc_init(struct device *arg)
 }
 
 SYS_INIT(soc_init, PRE_KERNEL_1, CONFIG_KERNEL_INIT_PRIORITY_DEFAULT);
+
+void z_platform_init(void)
+{
+	__asm__ volatile(
```

Comment:
Can you add a comment about what this asm code is trying to accomplish

Reply:
Done